i18n/en_US.yaml

```diff
@@ -516,8 +516,8 @@ ui:
     tag_wiki: tag wiki
     create_tag: Create Tag
     edit_tag: Edit Tag
-    ask_a_question: Add Question
-    edit_question: Edit Question
+    ask_a_question: Add question
```

Comment:
@Justman100 I don't think capital's here are grammar issues. just look at one of the paras i have taken from the internet.

![image](https://github.com/apache/incubator-answer/assets/9161543/fe10f2dd-5038-46fa-bae0-7673fa41eed3)

Just Wana say in English, each sentence stats with capital and that's **Not** the only place where we use captials. Also look an "English" word in previous sentence

Reply:
The point is that _Add Question_ also exists as an _Add question_ version!